concepts/maps/introduction.md

```diff
@@ -23,6 +23,4 @@ While there is no canonical format, choose a consistent way to represent the key
 
 ## Map module functions
 
-Elixir provides many functions for working with maps in the _Map module_. Some _Map module_ functions require an _anonymous_ or _captured function_ to be passed into the function to assist with the map transformation.
-
-Maps also implement the _Enumerable protocol_, which allow them to be used in _Enum module_ functions. Often maps are transformed into a list, processed or transformed in some way, then returned. When a map is returned as a list, each key-value pair is returned as a tuple in the list.
```

Comment:
Removing the mention of the Enumerable protocol because it's not relevant in this exercise. Enumerable and using `Enum` for maps is covered in the `enum` exercise (`boutique-inventory`)